### ISSUE TITLE
failed to set file label ... operation not supported

### ISSUE THREAD

**Issue by hevelius** [trajano/docker-volume-plugins#33]
Hi all,

any attempt to mount a volume (via compose file or cli mode) returns this error:

`docker: Error response from daemon: **failed to set file label** on /var/lib/docker/plugins/e3534cc8a654f232b07bd53094198ccd9b8039c17fc14964f2078d3e601a6c15/propagated-mount/6bfc1351bcbc76e90518cb94a70fad08a5b3a9f5a477a0a2eca86696e2ace1ce: **operation not supported**.
`

I'm using
Docker v19.03.8 in swarm mode
GlusterFS v4.1.9
on CentOS 7 with SELinux in Enforcing mode

Plugin configuration
`docker plugin install --alias glusterfs trajano/glusterfs-volume-plugin --grant-all-permissions --disable
docker plugin set glusterfs SERVERS=swarm-master1,swarm-master2,swarm-master3,swarm-worker1,swarm-worker2,swarm-worker3
docker plugin enable glusterfs`

Any ideas?

**Comment by trajano** [comment]
I haven't been maintaining this, but I am thinking it looks like an SELinux issue.

**Comment by hevelius** [comment]
Yes It could be an issue releated with SELinux and named volumes. For some reasons mounting storage using named volumes cause a re-labeling error. I tried to use Z flag and I've checked SELinux chcon on releated directories but not works.
